src/workerd/api/queue.c++

```diff
@@ -8,20 +8,17 @@
 namespace workerd::api {
 
 kj::StringPtr validateContentType(kj::StringPtr contentType) {
-  if (contentType == IncomingQueueMessage::ContentType::TEXT) {
+  if (toLower(contentType) == IncomingQueueMessage::ContentType::TEXT) {
```

Comment:
Nit, but it'd be less wasteful to just call `toLower` once at the start of the function rather than again in each `if`